tests/trafficmanagement/scripts/request_timeouts_delay.txt

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+# Copyright 2020 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source ${REPO_ROOT}/content/en/docs/tasks/traffic-management/request-timeouts/snips.sh
+
+# Export HOST, PORT and URL
+export INGRESS_HOST={{ .ingressHostCom }}
+export INGRESS_PORT={{ .ingressPortCom }}
+export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+
+# Step 1: add a half second request timeout for calls to the reviews service
+snip_request_timeouts_3
+
+istioctl experimental wait --for=distribution VirtualService reviews.default
```

Comment:
@frankbu Do you have any idea why this step fails. I keep getting following test failure.

```
TestRequestTimeouts: script.go:369: script scripts/request_timeouts.txt returned an error: exit status 1. Output:
        virtualservice.networking.istio.io/productpage created
        virtualservice.networking.istio.io/reviews created
        virtualservice.networking.istio.io/ratings created
        virtualservice.networking.istio.io/details created
        virtualservice.networking.istio.io/reviews configured
        Error: timeout expired before resource VirtualService/default/reviews became effective on all sidecars
```
I can see [traffic shifting test works fine with the same command](https://github.com/istio/istio.io/blob/master/tests/trafficmanagement/scripts/traffic_shifting.txt#L103). I'm I doing something wrong here ?

Reply:
Hmm, seems like it's just taking too long??? I'd try to add a delay before calling `istioctl wait` to see if that fixes it. Otherwise, maybe there's something different (wrong) with the environment setup in this vs the traffic-shifting test?

Reply:
I think I already tried adding `sleep` and `timeout` to wait for it but it didn't help.

```go
sleep 60s

istioctl experimental wait --for=distribution VirtualService ratings.default --timeout=60s
```

Reply:
Maybe there's something wrong with your setup in requests_timeouts_test.go? Maybe if you make the other changes I've suggested it will miraculously start working :)